src/Maker/MakeListener.php

```diff
@@ -102,6 +103,33 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
             $event = $io->askQuestion($question);
             $input->setArgument('event', $event);
         }
+
+        $event = $input->getArgument('event');
+        if (null === $this->getEventConstant($event) && null === $this->eventRegistry->getEventClassName($event)) {
+            $eventList = $this->eventRegistry->getAllActiveEvents();
+            $eventFQCNList = array_filter(array_map($this->eventRegistry->getEventClassName(...), $eventList), fn ($eventFQCN) => \is_string($eventFQCN));
+            $eventIdAndFQCNList = array_unique(array_merge($eventList, $eventFQCNList));
+            $suggestionList = [];
+            foreach ($eventIdAndFQCNList as $eventSuggestion) {
```

Comment:
Add spacing for readability between logic statements (e.g. `if`'s) to be consistent with other code in the bundle.

```suggestion
            $suggestionList = [];
            
            foreach ($eventIdAndFQCNList as $eventSuggestion) {
```

We need to do this for the other `if` statements below as well.